### PR TITLE
open source: what it means to me

### PR DESCRIPTION
**Pull request by reqshark**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -389,3 +389,32 @@ I am Tristan B. Kildaire or otherwise known as _**Deavmi**_. I founded an organi
...
+   awesomely arcane technique central to a program or system, esp. one
+   not generally published and available to hackers at large (compare
+   {black art}); one which could only have been composed by a true
+   {wizard}.  Compiler optimization techniques and many aspects of
+   {OS} design used to be {deep magic}; many techniques in
+   cryptography, signal processing, graphics, and AI still are.
+   Compare {heavy wizardry}.  Esp. found in comments of the form
+   "Deep magic begins here...".  Compare {voodoo programming}.
+
+```
```
Comment (nickdesaulniers) [inline README.md]: Did you mean to use `c` and `bash` syntax highlighting?  Are the `{{}}` meant to be `[link](url)`?  Is the whole quote from C.S.Lewis' Narnia books (esp the part about compiler optimization)?

Comment (reqshark) [inline README.md]: > Did you mean to use c and bash syntax highlighting?

yes.

> Is the whole quote from C.S.Lewis' Narnia books (esp the part about compiler optimization)?

No. Although [poss.] may be an indication `deep magic` *possibly* originates from C.S. Lewis.

Would you like me to document and link this material's true source?

Comment (nickdesaulniers) [inline README.md]: Maybe not the material's source, but I don't know what you mean by `compare voodoo programming.`  Idk what that is, or heavy wizardry.  Heavy metal, [yes](http://ironmaiden.com/).

Comment (reqshark) [inline README.md]: it's extracted from a historical document of the epoch behind open source, see https://github.com/reqshark/artifacts/blob/master/jarg262.txt

Comment (nickdesaulniers) [inline README.md]: ah, maybe just link to relevant parts of http://www.catb.org/jargon/

Comment (reqshark) [inline README.md]: oooo, that main jargon page is perfect: provides easy navigation and relelvant page linking surrounding the attributed source

**Comment by nickdesaulniers** [comment]
Cool, excellent references!  It's funny to see `<sub>` tags in markdown.  MD was supposed to be simpler than HTML, and yet is now being pervaded by...

Anyways thanks Bent! [Maybe you can help us find more contributors](https://github.com/nickdesaulniers/What-Open-Source-Means-To-Me/issues/20)?

**Comment by reqshark** [comment]
@ahdinosaur?

**Comment by justinclift** [comment]
@reqshark Bonus points for originality. :grin:

And some good tunes on SoundCloud too.  Listening to Cloud City... :wink:

**Comment by reqshark** [comment]
haha thanks @justinclift, that's a good track! :headphones:

**Comment by ahdinosaur** [comment]
thanks for the mention @reqshark, on my TODO. :snail:

**Pull request merged**